Containers/nextcloud/entrypoint.sh

```diff
@@ -37,6 +37,11 @@ if [ -n "$PHP_MAX_CHILDREN" ]; then
     sed -i "s/^pm.max_children =.*/pm.max_children = $PHP_MAX_CHILDREN/" /usr/local/etc/php-fpm.d/www.conf
 fi
 
+# Limit access to php-fpm
+# TODO
+grep -q ';listen.allowed_clients' /usr/local/etc/php-fpm.d/www.conf
+sed -i 's|;listen.allowed_clients.*|listen.allowed_clients = 127.0.0.1,::1#TODO|' /usr/local/etc/php-fpm.d/www.conf
```

Comment:
todo, see https://github.com/nextcloud/all-in-one/issues/3172#issue-1853054017

Reply:
And if we introduce dig we might also adjust the collabora logic to use dig insteatd of php